Bottom: 0c5e0c46d795ddb0d4d768c7123e594036a6a3bc
Top:    0423aa5b081921e258fa18c5c3ca6641a41660e8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:51:38 +0000

polygon1.c: Rename rule and v_rule to j_rule for consistency


---

diff --git a/src/polygon1.c b/src/polygon1.c
index ea2290c..6522512 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1550,7 +1550,7 @@ SubJ_Rule (char p, VNODE * v, DIRECTION * cdir)
  */
 /* *cur is considered a vertex */
 static int
-jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
+jump (VNODE ** cur, DIRECTION * cdir, J_Rule j_rule)
 {
   CVCList *d, *start;
   VNODE *e; /* e is considered an edge */
@@ -1580,7 +1580,7 @@ jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
       else
         e = VERTEX_FORWARD_EDGE (d->parent);
       newone = *cdir;
-      if (!e->Flags.mark && rule (d->poly, e, &newone))
+      if (!e->Flags.mark && j_rule (d->poly, e, &newone))
 	{
 	  if ((d->side == 'N' && newone == FORW) ||
 	      (d->side == 'P' && newone == BACKW))
@@ -1605,7 +1605,7 @@ jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
 
 /* start is considered a vertex */
 static int
-Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
+Gather (VNODE * start, PLINE ** result, J_Rule j_rule, DIRECTION initdir)
 {
   VNODE *cur = start, *newn; /* cur is considered a vertex */
   DIRECTION dir = initdir;
@@ -1616,7 +1616,7 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
   do
     {
       /* see where to go next */
-      if (!jump (&cur, &dir, v_rule))
+      if (!jump (&cur, &dir, j_rule))
 	break;
       /* add edge to polygon */
       if ((newn = poly_CreateNode (cur->point)) == NULL)
